.github/workflows/os-smoke-test.yml

```diff
@@ -25,14 +28,15 @@ jobs:
           java-version: '17'
           distribution: 'temurin'
           cache: 'maven'
+          check-latest: false
```

Comment:
:upside_down_face:  That's the default value: https://github.com/actions/setup-java#check-latest and temurin 17 should be in the tool cache for windows-2022: https://github.com/actions/virtual-environments/blob/main/images/win/Windows2022-Readme.md#java

Reply:
I was under the impression the latest was true :thinking: I guess I read too quickly, we can remove this then.